ts/components/staking/dashboard_hero.tsx

```diff
@@ -323,7 +334,10 @@ export const DashboardHero: React.FC<DashboardHeroProps> = ({
                                 {metrics.map(metric => {
                                     return (
                                         <Figure key={`${metric.title}${metric.number}`}>
-                                            <FigureTitle>{metric.title}</FigureTitle>
+                                            <FigureHeader>
+                                                <FigureTitle>{metric.title}</FigureTitle>
+                                                {metric.headerComponent && metric.headerComponent()}
```

Comment:
Copied the same pattern from the account page for consistency (in case you're wondering why the var name)